src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -87,30 +90,21 @@ static LinkedFile convert(List<String> entry) {
             }
         }
 
-        if (field == null) {
-            String pathStr = entry.get(1);
-            if (pathStr.contains("//")) {
-                // In case the path contains //, we assume it is a malformed URL, not a malformed path.
-                // On linux, the double slash would be converted to a single slash.
-                field = new LinkedFile(entry.get(0), pathStr, entry.get(2));
-            } else {
-                try {
-                    // there is no Path.isValidPath(String) method
-                    Path path = Path.of(pathStr);
-                    field = new LinkedFile(entry.get(0), path, entry.get(2));
-                } catch (InvalidPathException e) {
-                    field = new LinkedFile(entry.get(0), pathStr, entry.get(2));
-                }
+        try {
+            if (field == null) {
+                field = new LinkedFile(entry.get(0), Path.of(entry.get(1)), entry.get(2));
             }
-        }
 
-        // link is the only mandatory field
-        if (field.getDescription().isEmpty() && field.getLink().isEmpty() && !field.getFileType().isEmpty()) {
-            field = new LinkedFile("", Path.of(field.getFileType()), "");
-        } else if (!field.getDescription().isEmpty() && field.getLink().isEmpty() && field.getFileType().isEmpty()) {
-            field = new LinkedFile("", Path.of(field.getDescription()), "");
+            // link is only mandatory field
+            if (field.getDescription().isEmpty() && field.getLink().isEmpty() && !field.getFileType().isEmpty()) {
+                field = new LinkedFile("", Path.of(field.getFileType()), "");
+            } else if (!field.getDescription().isEmpty() && field.getLink().isEmpty() && field.getFileType().isEmpty()) {
+                field = new LinkedFile("", Path.of(field.getDescription()), "");
+            }
+            entry.clear();
+        } catch (InvalidPathException ignored) {
```

Comment:
This is sort of a quick hack. There is no loging happening, no output to the user, nothing.

I think, the invalid path is just thrown away. This should not happen. The invalid path should still be displayed in the UI. Possible, the path should be fixed. This should be easy in your case, because, the `file:///` can just be removed, can't it? There will be special handling required to handle the case `:/` when splitting the list for Linkedfile

(The issue is that x:file:///thing:pdf will be x, file, ///thing:pdf as list and not x, file:///thing, pdf. Please think of the parsing more instead of adding a quick fix. This will help many users manually editing the bibtex and hoping that JabRef does not kill data.